ts/utils/webview.ts

```diff
@@ -12,6 +12,62 @@ export const ON_SCROLL_END_LISTENER = `window.onscroll=function(){
     }
 };`;
 
+export const APP_EVENT_HANDLER = `
+function sendMessagesToRN(message) {
+  if (message) {
+    if (window.ReactNativeWebView) {
+      window.ReactNativeWebView.postMessage(JSON.stringify(message));
+    }
+    return;
+  }
+  console.error("Specify a message to send");
+}
+
+function closeModal() {
+  let message = {
```

Comment:
```suggestion
  const message = {
```
it is never updated, we could use const instead